### ISSUE TITLE
Moar Bugz & a Suggestion

### ISSUE THREAD

**Issue by Blowjobbilly** [rex706/SAM#81]
When attempting to copy passwords sam crashes.
Parameter options don't save.

**Comment by rex706** [comment]
Are you using the latest version? I don't experience any issues with copying account passwords through the context menu both with password protection enabled and disabled. 

I see that the -forceservice parameter is the only one that seems to not persist to the settings window. Is this what you are referencing? Or are there additional problems as well? Or do you mean that they do not apply to the Steam executable when opened through SAM? Could you please upload your SAMSettings.ini file? Make sure to remove the API key if you are using one.

**Comment by Blowjobbilly** [comment]
> Are you using the latest version? I don't experience any issues with copying account passwords through the context menu both with password protection enabled and disabled.
> 
> I see that the -forceservice parameter is the only one that seems to not persist to the settings window. Is this what you are referencing? Or are there additional problems as well? Or do you mean that they do not apply to the Steam executable when opened through SAM? Could you please upload your SAMSettings.ini file? Make sure to remove the API key if you are using one.


Yeah, I'm using the latest version. I updated through the updater.
But, idk it crashes for me.

![!](https://user-images.githubusercontent.com/58930346/71053770-3a77f480-2158-11ea-970b-f773e31eb3f8.gif)
I set only these few options, but they doesn't persist at all. As soon as I close the options menu they reset.

![!](https://user-images.githubusercontent.com/58930346/71053799-57acc300-2158-11ea-9bc9-56253563f340.gif)

And sure, here's the crash log and ini.
[logzz.zip](https://github.com/rex706/SAM/files/3976550/logzz.zip)

**Comment by rex706** [comment]
Been a while since I had the time to look back into this issue. Are you still having problems? It could potentially be related to needing [.NET Framework 4.8](https://dotnet.microsoft.com/download/dotnet-framework/thank-you/net48-web-installer). And I'm sorry you don't like the SAM "Metro" window theme, but I think at this point it would be too difficult to have it be a toggle-able setting.
